### DISCUSSION TITLE
【V5】请问如何实现x轴title横向展示，如果无法全部展示，就省略部分列的title

### DISCUSSION THREAD

**Discussion by PineSongCN** [antvis/G2#5244] [Q&A]
下图左边是G2，右边是ECharts；
另外动态修改宽高的情况下，legend的位置计算会出现错误；
![image](https://github.com/antvis/G2/assets/39976617/b27f3bb5-280c-443f-b8a2-92b875d170e4)

**Comment by PineSongCN** [comment]
@pearmini 你好，我在V4看到默认就是缩略列的显示，但是V5找不到配置的方式

**Comment by PineSongCN** [comment]
dd

**Comment by pearmini** [comment]
相同问题：https://github.com/antvis/G2/discussions/5285 ，这个先关闭了。
